### ISSUE TITLE
New Food Data Central API Released

### ISSUE THREAD

**Issue by littlebunch** [USDA/USDA-APIs#92]
A new version of the FDC API is now available.  This version includes new endpoints for requesting multiple foods in full or abridged formats, "list" endpoints for paged browsing of foods lists, the ability to request one or more nutrients plus much more. General documentation is available on the [FDC web site](https://fdc.nal.usda.gov/api-guide.html).  The OpenAPI v.3 specification is also available in [HTML](https://fdc.nal.usda.gov/fdc_api.html), [JSON](https://api.nal.usda.gov/fdc/v1?api_key=DEMO_KEY), and [YAML](https://api.nal.usda.gov/fdc/v1/yaml-spec?api_key=DEMO_KEY) formats.   

On a related note, the discontinuation date for the NDB API has been extended to May 31, 2020.

**Comment by manijcgvak** [comment]
As per the update, We observed there is the change in the Search API and the response get. Can u please describe what is the difference between the foodNutrients in Food Search endpoint and Food Details endpoint. We have updated our application from NDB to FDC recently when the API is changed. But now there is again an update available, when it will finalised So that we can alter our application.

**Comment by littlebunch** [comment]
@manijcgvak schema and endpoint details are documented [here](https://fdc.nal.usda.gov/fdc_api.html#/components/schemas/AbridgedFoodItem).  Basically, the search returns a list of AbridgedFoodNutrients while the detail endpoint returns either a list of FoodNutrient (the default) or a list of AbridgedFoodNutrients if you specify format=abridged.

I believe the current release is feature complete but I believe it won't be  officially finalized until April 30.  I can ask the program people to respond.

**Comment by KyleMcKillop-USDA** [comment]
That is correct, this version is now considered feature complete. Changes between now and April 30th are to address any discovered issues with this current release. Any further changes to the API will occur outside of the current feature set with new endpoints.

**Comment by manijcgvak** [comment]
As mentioned that after 30th Apr, API structure will not change. Please confirm on this so that we start work of top of the api's

**Comment by KyleMcKillop-USDA** [comment]
@manijcgvak We released an FDC software update this morning and should be our last software update until at least October.  Future updates to the API will not change the structure of version 1 endpoints.
